scripts/consensus-release.sh

```diff
@@ -258,6 +258,16 @@ else
     fi
 fi
 
+echo "Checking that the versions are consistent with the contents of the changelog directories"
 ./scripts/ci/check-consensus-release.sh
 
-echo "Succesfully created release. You should now inspect the current branch (${branch%?}) and if everything looks right, push it to GitHub, open a PR and get it merged!"
+echo "Checking that CHaP would be able to build these packages (this will take a while!)"
+./scripts/check-consensus-builds-with-chap.sh
+
+if [[ -z $? ]]; then
+  echo "Succesfully created release. You should now inspect the current branch (${branch%?}) and if everything looks right, push it to GitHub, open a PR and get it merged!"
+else
+  echo "Your release seem to depend on changes that are not visible in CHaP/Hackage"
+  echo "Maybe you added a function to a separate package or are using a package that is not in those indices?"
+  echo "I created the branch ${branch%?} but it is probably wrong, so you should delete it, check the error above and re-run this script. I will leave the branch just in case."
```

Comment:
Is this line a copy-paste error?

Reply:
I don't know specifically what you mean. But my intention was to get that message on the screen, having the script talking as "I" :smile:

Reply:
Ah, OK. That seemed odd to me in the PR diff, but I don't think it would confuse me on the command-line output.